### ISSUE TITLE
[SDK2013] [Feature Request] Move to 64-Bit and increase edict limit from 2048 to 8192

### ISSUE THREAD

**Issue by dangreene0** [ValveSoftware/source-sdk-2013#584]
Increasing the edict limit would solve a lot of issues for many multiplayer mods using SDK2013 as a base. Increasing this limit and allowing for a 64bit branch would allow games to better utilize modern hardware.

Kind of a copy of ValveSoftware/Source-1-Games#5447 since it would benefit both games.

**Comment by ikir83** [comment]
Yes I made a mod for Insurgency 2014 and I needed to lower edict number via server config in some maps to avoid crashes

**Comment by sour-dani** [comment]
> Since my request was closed due to being a duplicate of this one ill add my thoughts here.
> 
> Raising the edict limit out of nowhere is a bit dangerous and can break things for some mods. A better solution to this would be allowing the modder to control it. So I suggest adding an entry to the gameinfo.txt that controls the edict limit. This can be defaulted to 2048 for mods that do not want to use it or don't have it in their gameinfo.txt. This feature is exactly like how Goldsrc lets you control the edict limit in its liblist.gam file. https://developer.valvesoftware.com/wiki/Liblist.gam
> 
> There are problems with this however as it will require changes to the hard coded MAX_EDICTS define in public/const.h. This can probably be solved by adding a new function to the engine interface that will grab the current edict limit. Any older mods that don't use this will be completely fine as they will still be using the old hard coded 2048 limit which will be the default.

Sound compromise.
